src/xSTIR/pSTIR/STIR.py

```diff
@@ -1627,22 +1627,40 @@ def get_time_at_which_num_prompts_exceeds_threshold(self, threshold):
 
 class PoissonNoiseGenerator(object):
     """
-    Class that generates Poisson noise.
+    Generates noise realisations according to Poisson statistics but allowing for scaling.
+
+    A scaling_factor is used to multiply the input data before generating
+    the Poisson random number. This means that a scaling_factor larger than 1
+    will result in less noisy data.
```

Comment:
let's be explicit (STIR isn't) and say ""will result in data with lower relative noise"